Source/Core/Common/JitRegister.h

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 
 #pragma once
+#include <fmt/format.h>
 #include <stdarg.h>
 #include <string>
 #include "Common/CommonTypes.h"
```

Comment:
Although this is fine, I think it'd be better to order it like this (and `stdarg.h` is no longer needed):

```C++
#pragma once

#include <string>

#include <fmt/format.h>

#include "Common/CommonTypes.h"
```